fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java

```diff
@@ -56,19 +56,25 @@
      */
 
     public PropertiesRdfContext(final FedoraResource resource,
-                                final IdentifierConverter<Resource, FedoraResource> idTranslator)
```

Comment:
line 59? It was formatted by Eclipse using the auto style check using the https://github.com/fcrepo4/fcrepo4/tree/master/src/site/eclipse and I run mvn checkstyle:check before submit PR.